insights/core/spec_factory.py

```diff
@@ -156,41 +156,6 @@ def __repr__(self):
         return '%s("%r")' % (self.__class__.__name__, self.path)
 
 
-class MetadataProvider(FileProvider):
-    """
-    Class used for insights-core built-in files.  These files should not
-    be filtered, redacted or blocked.
```

Comment:
@xiangce do you need to include some logic to make sure that the `tags` datasources are not redacted?

Reply:
@bfahr - With the new implementation (in #3679), these  built-in files will be excluded explicitly [here](https://github.com/RedHatInsights/insights-core/blob/2beef363db473b88221d590aea1df97194d6b150/insights/core/spec_cleaner.py#L44)

And in further, they can also be excluded by adding the `not_obfuscate` option [here](https://github.com/RedHatInsights/insights-core/pull/3679/files#diff-11e9e8667252cdaf31413d23bdfbe835b1b0eed3a307700ff69a4266e72b472dR5-R11)

Please let me know if this work for you on this point.